{"name": "test_run_invalid_api_wrapper", "status": "passed", "description": "Test that _run fails gracefully when `api_wrapper.run` is not implemented correctly.", "start": 1743501667874, "stop": 1743501667875, "uuid": "96f950fd-a2aa-4da9-9dc8-fb60e56d3ee2", "historyId": "0cfd6ac5aa753ad68889f8c0c1757507", "testCaseId": "0cfd6ac5aa753ad68889f8c0c1757507", "fullName": "src.tests.test_base_tool.TestBaseAction#test_run_invalid_api_wrapper", "labels": [{"name": "tag", "value": "negative"}, {"name": "tag", "value": "base"}, {"name": "tag", "value": "unit"}, {"name": "parentSuite", "value": "src.tests"}, {"name": "suite", "value": "test_base_tool"}, {"name": "subSuite", "value": "TestBaseAction"}, {"name": "host", "value": "fv-az2032-69"}, {"name": "thread", "value": "2321-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.test_base_tool"}]}